### PR TITLE
Translate Testing Recipes

### PR DESCRIPTION
**Pull request by winndo**
Hi, please review indonesian translation for page [testing-recipes](https://reactjs.org/docs/testing-recipes.html)

### CODE REVIEW

File: content/docs/testing-recipes.md [hunk 1]
```diff
@@ -416,17 +416,17 @@ it("changes value when clicked", () => {
 });
 ```
 
-Different DOM events and their properties are described in [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent). Note that you need to pass `{ bubbles: true }` in each event you create for it to reach the React listener because React automatically delegates events to the document.
+*DOM events* yang berbeda dan properti-properti mereka dijabarkan di [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent). Perlu dicatat bahwa anda perlu mengoper `{ bubbles: true }` pada setiap *event* yang anda buat agar *event* tersebut dapat mencapai React *listener* karena React secara otomatis mendelegasikan *event-event*  itu ke dokumen.
```
Comment (resir014) [inline content/docs/testing-recipes.md]:
```suggestion
*Event* DOM yang berbeda dan properti-properti mereka dijabarkan di [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent). Perlu dicatat bahwa anda perlu mengoper `{ bubbles: true }` pada setiap *event* yang anda buat agar *event* tersebut dapat mencapai *listener* React karena React secara otomatis mendelegasikan *event-event*  itu ke dokumen.
```

File: content/docs/testing-recipes.md [hunk 2]
```diff
@@ -416,17 +416,17 @@ it("changes value when clicked", () => {
...
 
-> Note:
+Catatan:
 >
-> React Testing Library offers a [more concise helper](https://testing-library.com/docs/dom-testing-library/api-events) for firing events.
+> *React Testing Library* menawarkan [bantuan-bantuan yang lebih ringkas](https://testing-library.com/docs/dom-testing-library/api-events) untuk melepaskan *event-event*.
```
Comment (resir014) [inline content/docs/testing-recipes.md]:
```suggestion
> React Testing Library menawarkan [bantuan-bantuan yang lebih ringkas](https://testing-library.com/docs/dom-testing-library/api-events) untuk melepaskan *event-event*.
```

Similar as [this](https://twitter.com/ivanlanin/status/898520073224167424), this is the name for a library/framework.

File: content/docs/testing-recipes.md [hunk 3]
```diff
@@ -537,15 +537,15 @@ it("should accept selections", () => {
...
 ---
 
-### Snapshot Testing {#snapshot-testing}
+### Pengujian *Snapshot* {#snapshot-testing}
 
-Frameworks like Jest also let you save "snapshots" of data with [`toMatchSnapshot` / `toMatchInlineSnapshot`](https://jestjs.io/docs/en/snapshot-testing). With these, we can "save" the rendered component output and ensure that a change to it has to be explicitly committed as a change to the snapshot.
+*Framework* seperti Jest juga dapat menyimpan *"snapshots"* data dengan [`toMatchSnapshot` / `toMatchInlineSnapshot`](https://jestjs.io/docs/en/snapshot-testing). Dengan ini, kita dapat "menyimpan" keluaran komponen yang sudah ter-*render* dan memastikan perubahan pada komponen itu komit secara eksplisit seperti perubahan pada *snapshot*.
 
-In this example, we render a component and format the rendered HTML with the [`pretty`](https://www.npmjs.com/package/pretty) package, before saving it as an inline snapshot:
+Pada contoh ini, kita akan melakukan *render* komponen dan membentuk hasil *render HTML* dengan paket [`pretty`](https://www.npmjs.com/package/pretty), sebelum menyimpan sebagai *snapshot inline*:
```
Comment (resir014) [inline content/docs/testing-recipes.md]:
```suggestion
Pada contoh ini, kita akan melakukan *render* komponen dan membentuk hasil *render* HTML dengan paket [`pretty`](https://www.npmjs.com/package/pretty), sebelum menyimpan sebagai *snapshot inline*:
```

**Comment by hantumobil** [comment]
Hi @resir014 , thanks for detailed review. I have bundle fixed typo and abbreviations based on your suggestion. please take a look.

File: content/docs/testing-recipes.md [hunk 4]
```diff
@@ -337,9 +337,9 @@ it("should render contact information", () => {
 
 ---
 
-### Events {#events}
+### *Events* {#events}
 
-We recommend dispatching real DOM events on DOM elements, and then asserting on the result. Consider a `Toggle` component:
+Kami menyarankan pengiriman *DOM events* sebenarnya dari elemen *DOM*, lalu melakukan *asserting* pada hasilnya. Dapat dilihat pada komponen `Toggle` berikut:
```
[the remark this answers is not in the document]
Comment (hantumobil) [reply content/docs/testing-recipes.md]: Okay, fixed

File: content/docs/testing-recipes.md (hunk 1 above)
Comment (hantumobil) [reply content/docs/testing-recipes.md]: change all *DOM events* to *Event* DOM

**Comment by winndo** [comment]
need to inform you @resir014 that I've change my username from @hantumobil to @winndo, thanks!

**Pull request merged**